src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -744,6 +753,34 @@ private void AddExecutablePathCommand(IList<LaunchCommand> commands)
             commands.Add(new LaunchCommand("-file-exec-and-symbols " + exe, description, ignoreFailures: false, failureHandler: failureHandler));
         }
 
+        private void DetermineAndAddExecutablePathCommand(IList<LaunchCommand> commands)
+        {
+            // Runs a shell command to get the full path of the exe.
+            string absoluteExePath = string.Format(CultureInfo.InvariantCulture, @"shell readlink -f /proc/{0}/exe", _launchOptions.ProcessId);
```

Comment:
Something like:
shell [ $(uname -s) == "Linux" ] && { readlink -f /proc/{0}/exe; true; } || echo "Error: Unsupported operating system family '$(uname -s)'"

---
In reply to: [75935390](https://github.com/Microsoft/MIEngine/pull/422#discussion_r75935390) [](ancestors = 75935390)